app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -69,7 +69,11 @@ class BrowserToolbarView(
 
             customView.copy.setOnClickListener {
                 popupWindow.dismiss()
-                clipboard.text = view.url.toString()
+                if (isCustomTabSession) {
+                    clipboard.text = customTabSession?.url
+                } else {
+                    clipboard.text = view.url.toString()
```

Comment:
There are some conversions around not showing the protocol (HTTP or HTTPS) for saving space on the toolbar, this may break in the future. Is it possible to use the `url` from the session?

Reply:
As far as I have tested, If you have Fenix already opened, using `selectedSession` will return the url from that tab.

Reply:
But the conversion you mention is just for the customTab `view.url,` `customTabSession` still retains the full url.

Reply:
Yes, but I'm referring to the normal tabs case, where we are using the `url` directly from the view `clipboard.text = view.url.toString()`. If this view doesn't include the protocol it will happen something similar to what we are fixing with custom tabs now

Reply:
I understand now. I was focused only on my own changes. :)
I will change that too, to get url from the session. Thanks for the heads=up! 👍

Reply:
👍